style/javascript.md

```diff
@@ -536,6 +537,404 @@ to replace all uses of `var` in existing files.
 
 
 -----------------
+### Move off Underscore
+
+Now that we're using ES6/7 many of the features of Underscore.js are now built in to the language! We should move away from using Underscore in our code towards making use of these native features.
+
+There are a couple methods that are sufficiently complicated and don't have a direct equivalent so instead we have a custom-built copy of lodash containing only those methods. You can find this file at: `third_party/javascript-khansrc/lodash/lodash.js` along with instructions on how to build it and exactly what methods are included.
+
+What follows is a method-by-method set of equivalents for what we've used in Underscore and what you should be using in ES6/7 instead:
+
+#### _.bind
+
+```js
+_.bind(fn, someObj, args)
+```
+
+Becomes:
+```js
+fn.bind(someObj, args)
+```
+
+Note that cases where you had:
+
+```js
+_.bind(function() { ... }, this)
+```
+
+should instead become:
+
+```js
+() => { ... }
```

Comment:
With args, do we prefer `(...args) => { someFn(...args) }`? There was some talk about this the other day.